addons/udes_stock/models/product_product.py

```diff
@@ -11,3 +10,29 @@ class ProductProduct(models.Model):
 
     # Add tracking for archiving.
     active = fields.Boolean(tracking=True)
+
+    def sync_active_to_templates(self, activate_templates=True, deactivate_templates=True):
```

Comment:
We've added this method but it isn't being called anywhere, is it needed for this story?

Reply:
This is called in the `execute` method inside `UDESItemMasterRecord`.